Incorrect content of the build/_maven_overlay directory causes the build project failure

### What happened?

For some reason, my `build/_maven_overlay` directory had some leftovers from some previous and old build which ended up being included in the generated operator images and causes the failure of the build as there were multiple version of the logback library. 

The `maven-overlay` target in the `Makefile` or the overlay generator script should cleanup `build/_maven_overlay`  before copying the content from `java/maven-logging`

### Steps to reproduce

_No response_

### Relevant log output

```shell
{"level":"info","ts":"2024-05-07T09:50:20Z","logger":"camel-k.controller.build","msg":"State transition","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"Build","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0","phase-from":"Scheduling","phase-to":"Pending"}
{"level":"info","ts":"2024-05-07T09:50:20Z","logger":"camel-k.controller.build","msg":"Build queue duration 973.429847ms","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"master","build-queue-duration":0.973429847,"api-version":"camel.apache.org/v1","kind":"Build","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0"}
{"level":"info","ts":"2024-05-07T09:50:20Z","logger":"camel-k.controller.integrationkit","msg":"Invoking action build","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"IntegrationKit","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0"}
{"level":"info","ts":"2024-05-07T09:50:20Z","logger":"camel-k.controller.build","msg":"State transition","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"Build","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0","phase-from":"Pending","phase-to":"Running"}
{"level":"info","ts":"2024-05-07T09:50:20Z","logger":"camel-k.builder","msg":"running builder task builder in context directory: /tmp/kit-cosvhigggors73cbqde0-1903413014"}
{"level":"info","ts":"2024-05-07T09:50:20Z","logger":"camel-k.controller.integrationkit","msg":"Invoking action build","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"IntegrationKit","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0"}
{"level":"info","ts":"2024-05-07T09:50:20Z","logger":"camel-k.controller.integrationkit","msg":"Build running","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"IntegrationKit","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0"}
{"level":"info","ts":"2024-05-07T09:50:20Z","logger":"camel-k.maven.build","msg":"Executed command: /usr/bin/cp --recursive /usr/share/maven/mvnw/. ."}
{"level":"info","ts":"2024-05-07T09:50:20Z","logger":"camel-k.maven","msg":"executing: ./mvnw -V --no-transfer-progress -Dstyle.color=never package -Dmaven.repo.local=/etc/maven/m2 --global-settings /tmp/kit-cosvhigggors73cbqde0-1903413014/maven/settings.xml -Dmaven.artifact.threads=32 -T 32","MAVEN_OPTS":""}
{"level":"info","ts":"2024-05-07T09:50:20Z","logger":"camel-k.maven.build","msg":"Executed command: ./mvnw -V --no-transfer-progress -Dstyle.color=never package -Dmaven.repo.local=/etc/maven/m2 --global-settings /tmp/kit-cosvhigggors73cbqde0-1903413014/maven/settings.xml -Dmaven.artifact.threads=32 -T 32"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"---------------------------------------------------"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[0]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/conf/logging/"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[1]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-embedder-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[2]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-settings-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[3]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-settings-builder-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[4]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-plugin-api-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[5]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-model-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[6]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-model-builder-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[7]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-builder-support-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[8]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-resolver-api-1.6.3.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[9]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-resolver-util-1.6.3.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[10]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-shared-utils-3.3.4.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[11]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/commons-io-2.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[12]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/guice-4.2.2-no_aop.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[13]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/guava-25.1-android.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[14]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/javax.inject-1.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[15]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/javax.annotation-api-1.2.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[16]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/plexus-utils-3.3.1.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[17]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/plexus-sec-dispatcher-2.0.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[18]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/plexus-cipher-2.0.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[19]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/commons-lang3-3.8.1.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[20]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-core-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[21]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-repository-metadata-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[22]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-artifact-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[23]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-resolver-provider-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[24]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-resolver-impl-1.6.3.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[25]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-resolver-spi-1.6.3.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[26]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/org.eclipse.sisu.inject-0.3.5.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[27]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/plexus-interpolation-1.26.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[28]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/plexus-component-annotations-2.1.0.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[29]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-compat-3.8.6.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[30]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/wagon-provider-api-3.5.1.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[31]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/org.eclipse.sisu.plexus-0.3.5.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[32]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/commons-cli-1.4.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[33]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/wagon-http-3.5.1-shaded.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[34]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/jcl-over-slf4j-1.7.36.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[35]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/wagon-file-3.5.1.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[36]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-resolver-connector-basic-1.6.3.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[37]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/maven-resolver-transport-wagon-1.6.3.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[38]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/jansi-2.4.0.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[39]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/jackson-databind-2.9.10.8.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[40]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/logback-classic-1.2.11.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[41]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/logback-classic-1.2.3.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[42]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/jackson-annotations-2.14.1.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[43]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/jackson-annotations-2.15.2.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[44]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/jackson-annotations-2.9.10.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[45]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/jackson-core-2.14.1.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[46]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/jackson-core-2.15.2.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[47]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/jackson-core-2.9.10.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[48]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/jackson-databind-2.14.1.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[49]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/jackson-databind-2.15.2.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[50]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/logback-classic-1.3.12.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[51]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/logback-core-1.2.11.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[52]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/logback-core-1.2.3.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[53]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/logback-core-1.3.12.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[54]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/logstash-logback-encoder-4.11.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[55]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/logstash-logback-encoder-7.2.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[56]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/logstash-logback-encoder-7.4.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"constituent[57]: file:/usr/share/maven/wrapper/dists/apache-maven-3.8.6-bin/5f1464e3/apache-maven-3.8.6/lib/slf4j-api-2.0.7.jar"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"---------------------------------------------------"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"Exception in thread \"main\" java.lang.reflect.InvocationTargetException"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.apache.maven.wrapper.BootstrapMainStarter.start(BootstrapMainStarter.java:53)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.apache.maven.wrapper.WrapperExecutor.execute(WrapperExecutor.java:152)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.apache.maven.wrapper.MavenWrapperMain.main(MavenWrapperMain.java:76)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"Caused by: java.lang.NoSuchMethodError: 'void ch.qos.logback.classic.LoggerContext.setMDCAdapter(org.slf4j.spi.MDCAdapter)'"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat ch.qos.logback.classic.spi.LogbackServiceProvider.initialize(LogbackServiceProvider.java:46)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.slf4j.LoggerFactory.bind(LoggerFactory.java:183)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.slf4j.LoggerFactory.performInitialization(LoggerFactory.java:170)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.slf4j.LoggerFactory.getProvider(LoggerFactory.java:455)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.slf4j.LoggerFactory.getILoggerFactory(LoggerFactory.java:441)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.apache.maven.cli.MavenCli.logging(MavenCli.java:497)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.apache.maven.cli.MavenCli.doMain(MavenCli.java:283)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.apache.maven.cli.MavenCli.main(MavenCli.java:196)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\tat org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.maven.build","msg":"\t... 7 more"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.builder","msg":"step failed with error: failure while building project: : exit status 1","step":"github.com/apache/camel-k/v2/pkg/builder/BuildQuarkusRunner","phase":"20","task":"builder"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.controller.build","msg":"Build duration 1.298038979s","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"master","build-attempt":5,"build-result":"Error","build-duration":1.298038979,"api-version":"camel.apache.org/v1","kind":"Build","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0"}
{"level":"error","ts":"2024-05-07T09:50:21Z","logger":"camel-k.controller.build","msg":"Build kit-cosvhigggors73cbqde0 failed: failure while building project: : exit status 1","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"Build","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0","stacktrace":"github.com/apache/camel-k/v2/pkg/util/log.Logger.Errorf\n\tgithub.com/apache/camel-k/v2/pkg/util/log/log.go:66\ngithub.com/apache/camel-k/v2/pkg/controller/build.(*monitorRoutineAction).updateBuildStatus\n\tgithub.com/apache/camel-k/v2/pkg/controller/build/monitor_routine.go:216\ngithub.com/apache/camel-k/v2/pkg/controller/build.(*monitorRoutineAction).runBuild\n\tgithub.com/apache/camel-k/v2/pkg/controller/build/monitor_routine.go:200"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.controller.build","msg":"State transition","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"Build","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0","phase-from":"Running","phase-to":"Failed"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.controller.integrationkit","msg":"Invoking action build","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"IntegrationKit","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.controller.build","msg":"State transition","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"Build","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0","phase-from":"Failed","phase-to":"Error"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.controller.build","msg":"Build error","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"Build","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0","reason":"failure while building project: : exit status 1","error-message":"failure while building project: : exit status 1"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.controller.integrationkit","msg":"Invoking action build","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"IntegrationKit","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.controller.integrationkit","msg":"State transition","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"IntegrationKit","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0","phase-from":"Build Running","phase-to":"Error"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k","msg":"Kit kit-cosvhigggors73cbqde0 ready, notify integration: master"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.controller.integrationkit","msg":"Invoking action error","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"kit-cosvhigggors73cbqde0","api-version":"camel.apache.org/v1","kind":"IntegrationKit","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"kit-cosvhigggors73cbqde0"}
{"level":"info","ts":"2024-05-07T09:50:21Z","logger":"camel-k.controller.integration","msg":"State transition","request-namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","request-name":"master","api-version":"camel.apache.org/v1","kind":"Integration","ns":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"master","phase-from":"Building Kit","phase-to":"Error"}
{"level":"error","ts":"2024-05-07T09:50:21Z","msg":"Reconciler error","controller":"integration-controller","controllerGroup":"camel.apache.org","controllerKind":"Integration","Integration":{"name":"master","namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c"},"namespace":"test-225e0926-e0e1-4a8b-a470-4619f598176c","name":"master","reconcileID":"80c92b9c-c1d3-4b84-b7b5-6772ec533f67","error":"error during trait customization: mount trait execution failed: unable to find integration container: master","stacktrace":"sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler\n\tsigs.k8s.io/controller-runtime@v0.16.5/pkg/internal/controller/controller.go:329\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem\n\tsigs.k8s.io/controller-runtime@v0.16.5/pkg/internal/controller/controller.go:266\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2\n\tsigs.k8s.io/controller-runtime@v0.16.5/pkg/internal/controller/controller.go:227"}
```
```


### Camel K version

main

--- a/script/Makefile
+++ b/script/Makefile
@@ -433,6 +433,7 @@ check-licenses:
 # The below are dependencies needed for maven structured logs. We must bundle into the final container image.
 maven-overlay:
 	@echo "####### Preparing maven dependencies bundle..."
+	rm -rf build/_maven_overlay
 	mkdir -p build/_maven_overlay
 	./script/maven_overlay.sh build/_maven_overlay
 

